git/githistory/rewriter.go

```diff
@@ -253,6 +267,13 @@ func (r *Rewriter) Rewrite(opt *RewriteOptions) ([]byte, error) {
 			if err != nil {
 				return nil, err
 			}
+			if len(opt.ObjectMapFilePath) > 0 {
+				mapStr := fmt.Sprintf("%s,%s\n", hex.EncodeToString(oid), hex.EncodeToString(newSha))
```

Comment:
Could this line be changed to the following:

```go
if _, err := fmt.Fprintf("%x,%x\n", oid, newSha); err != nil {
        return nil, err
}
```

for brevity?

Reply:
I have change it